pkg/cloud/services/ec2/vpc.go

```diff
@@ -56,6 +58,11 @@ func (s *Service) reconcileVPC() error {
 		return nil
 	}
 
+	// Make sure attributes are configured
+	if err := s.ensureManagedVPCAttributes(vpc); err != nil {
+		return errors.Wrapf(err, "failed to to set vpc attributes for %q", vpc.ID)
+	}
+
 	// Make sure tags are up to date.
 	err = tags.Ensure(vpc.Tags, &tags.ApplyParams{
 		EC2Client:   s.scope.EC2,
```

Comment:
missing failed event for ensuring tags?

Reply:
This could be a no-op though (like your similar comments above). How about I add it in `createVPC` for the `tags.Apply` command?

Reply:
Success could be a noop, but failure should actually be the result of an error from AWS

Reply:
Sorry, I misread your comment